cosim/include/verilator/bp_zynq_pl.h

```diff
@@ -103,12 +103,10 @@ class bp_zynq_pl {
 #ifdef GP1_ENABLE
     axi_gp1 = std::make_unique<axilm<GP1_ADDR_WIDTH, GP1_DATA_WIDTH> >(
         STRINGIFY(GP1_HIER_BASE));
-    axi_gp1->reset(tick);
```

Comment:
Oh is this not necessary anymore? Why not?

Reply:
Just kidding. They are added back now.

Reply:
:)